contrib/gitian-descriptors/main-build/galaxy-linux-script

```diff
@@ -45,8 +45,8 @@ export OUTDIR
   CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --disable-gui-tests"
   # CONFIGFLAGS=""
 
-  FAKETIME_HOST_PROGS="ar ranlib nm ld ldd"
-  FAKETIME_PROGS="date ar ranlib nm ld ldd"
+FAKETIME_HOST_PROGS="ar ranlib nm ld ldd"
+FAKETIME_PROGS="date ar ranlib nm ld ldd tar gzip"
 
```

Comment:
Not needed change of indent (whitespace).
Please fix it in separate PR when you can.